base_conditional_image/__manifest__.py

```diff
@@ -9,7 +9,7 @@
     "license": "AGPL-3",
     "category": "Misc",
     "depends": [
-        "base",
+        "mail",
```

Comment:
This change deserves a comment if we go this way, no?

Reply:
Yes, at least... However, I'd prefer to something like this instead https://github.com/OCA/web-api/pull/3/files#diff-5d31acaa664c7c8d9dd2b526e6f5acf160e25fd3cd837db900de3e6657bb5b68